packages/webviz-core/src/loadWebviz.js

```diff
@@ -178,7 +178,6 @@ const defaultHooks = {
     const { REMOTE_BAG_URL_2_QUERY_KEY } = require("webviz-core/src/util/globalConstants");
     return [REMOTE_BAG_URL_2_QUERY_KEY];
   },
-  maybeUpdateURLToTrackLayout: () => {},
```

Comment:
maybe just have this return false? It's a bit weird to use this value below knowing full well it doesn't exist.